var/spack/repos/builtin/packages/yafyaml/package.py

```diff
@@ -25,6 +25,8 @@ class Yafyaml(CMakePackage):
 
     version("main", branch="main")
 
+    # Generate checksum with spack checksum yafyaml@x.y.z
```

Comment:
Can we remove these comments?

Reply:
@alalazo I guess so, I just use them to remind myself how to do it. I'll clean them out when I get home.

That said, I swear in the past I could do `spack checksum package` and I'd get lots of versions and spack would ask how many to checksum. But now no matter what, I seem to only get one. For example:

```
❯ spack checksum gcc
==> Found 26 versions of gcc:

  12.2.0  https://ftp.gnu.org/gnu/gcc/gcc-12.2.0/gcc-12.2.0.tar.xz
  12.1.0  https://ftp.gnu.org/gnu/gcc/gcc-12.1.0/gcc-12.1.0.tar.xz
  11.3.0  https://ftp.gnu.org/gnu/gcc/gcc-11.3.0/gcc-11.3.0.tar.xz
  11.2.0  https://ftp.gnu.org/gnu/gcc/gcc-11.2.0/gcc-11.2.0.tar.xz
  11.1.0  https://ftp.gnu.org/gnu/gcc/gcc-11.1.0/gcc-11.1.0.tar.xz
  10.4.0  https://ftp.gnu.org/gnu/gcc/gcc-10.4.0/gcc-10.4.0.tar.xz
  10.3.0  https://ftp.gnu.org/gnu/gcc/gcc-10.3.0/gcc-10.3.0.tar.xz
  10.2.0  https://ftp.gnu.org/gnu/gcc/gcc-10.2.0/gcc-10.2.0.tar.xz
  10.1.0  https://ftp.gnu.org/gnu/gcc/gcc-10.1.0/gcc-10.1.0.tar.xz
  ...
  5.5.0   https://ftp.gnu.org/gnu/gcc/gcc-5.5.0/gcc-5.5.0.tar.xz

==> How many would you like to checksum? (default is 1, q to abort)
```
but:
```
❯ spack checksum fargparse
==> Found 1 version of fargparse:

  1.4.1  https://github.com/Goddard-Fortran-Ecosystem/fArgParse/archive/refs/tags/v1.4.1.tar.gz

==> Fetching https://github.com/Goddard-Fortran-Ecosystem/fArgParse/archive/refs/tags/v1.4.1.tar.gz

    version("1.4.1", sha256="8f9b92a80f05b0a8ab2dd5cd309ad165041c7fcdd589b96bf75c7dd889b9b584")
```
yet there are [quite a few versions on GitHub](https://github.com/Goddard-Fortran-Ecosystem/fArgParse/releases).

Do you know what I've screwed up in my `package.py` here to not enable the multi-checksum?